src/index.html

```diff
@@ -22,6 +22,349 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a
+          class="logo"
+          href="index.html"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="moyo-logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#section-1"
+                class="nav__link is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-2"
+                class="nav__link"
+              >
+                SAMSUNG
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-3"
+                class="nav__link"
+              >
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-4"
+                data-qa="nav-hover"
+                class="nav__link laptops"
```

Comment:
Avoid using additional classes for state or modifiers directly on elements like 'laptops'. Instead, use BEM modifiers such as 'nav__link--laptops' if it's a modifier or create a new BEM block if it represents a different component.